Stylesheets/Components/avatar.css

```diff
@@ -0,0 +1,48 @@
+.avatar-circle {
+    max-width: 100%;
+    height: auto;
+    border-radius: 50%;
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+.avatar-square {
+    max-width: 100%;
+    height: auto;
+    border-radius: 10%;
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+.avatar img {
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+.avatar-shadow {
+    box-shadow: 10px 10px 2px 10px #888888;
+}
+
+.sm {
+    width: 1.5rem;
+    height: 1.5rem;
+
+}
+
+.md {
+    width: 2.5rem;
+    height: 2.5rem;
+
+}
+
+.lg {
+    width: 3.5rem;
+    height: 3.5rem;
+
+}
+
+.shadow {
+    box-shadow: 3px 3px 3px 3px rgba(49, 47, 47, 0.2);
+    ;
```

Comment:
You've got an extra semi-colon here

Reply:
Hi vanshita,
Thanks for pointing it out I have removed it.